nominees/vxnaid.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Vxnaid",
+  "aliases": [
+    "Vaccination Monitoring Platform"
+  ],
+  "description": "Vxnaid is a digital platform that supports correct vaccine administration & efficient vaccination campaigns. Enable broad adoption of Vxnaid especially in RLS countries to ensure correct administration of vaccines, identification of vaccinees, follow-up/recall of vaccinated persons and understanding of vaccination campaign effectiveness.",
+  "website": "https://healthforhumanityreport.jnj.com/",
```

Comment:
I don't see any reference to Vxnaid on this site/page. Could we get a link to a site that has more information about this project?

Reply:
Fixed in commit e1f04d6